.github/validation_grid_utils.py

```diff
@@ -48,6 +50,7 @@ def cache_key() -> None:
 @app.command(name="download")
 def download_validation_grids() -> None:
     """Effectively download the validation grid files."""
+    config.TEST_DATA_PATH = pathlib.Path(os.getcwd()) / definitions.DEFAULT_TEST_DATA_FOLDER
```

Comment:
here is the main issue: `uv run` of this script installs `icon4py.model.testing` in some uv cache directory (for run isolation), but the default path for grid data is relative to the location of some file in icon4py.model.testing.